recipes/cymetric/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "1.5.0" %}
+{% set org = "cyclus" %}
+{% set branch = "1.5.0" %}
+{% set sha256 = "b1c60e68d9ee4421fbdeebf768a1ac48a31fc5a3559583b3f79e3c9b0aa4bb50" %}
+
+package:
+  name: cymetric
+  version: {{ version }}
+
+source:
+  fn: cymetric-{{ version }}-{{ sha256 }}.tar.gz
+  url: https://github.com/{{ org }}/cymetric/archive/{{ branch }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or py34]
+
+build:
+  number: 0
+  script: python setup.py install --prefix ${PREFIX}
```

Comment:
should this also have `--single-version-externally-managed --record record.txt`?